doc/man7/provider.pod

```diff
@@ -14,6 +14,8 @@ provider - OpenSSL operation implementation providers
 
 =head2 General
 
+This page contains information useful to authors of OpenSSL providers.
```

Comment:
Please change `to authors of OpenSSL providers` back to `for provider authors`.  `OpenSSL providers` can be understood to mean *for authors of providers distributed by OpenSSL*, which may deter any external provider author.